services/QuillLMS/app/models/activity_session.rb

```diff
@@ -66,6 +66,7 @@ class ActivitySession < ActiveRecord::Base
   before_save   :set_completed_at, :set_activity_id
 
   after_save    :determine_if_final_score, :update_milestones
+  after_save :record_teacher_activity_feed, if: [:completed_at_changed?, :completed?]
```

Comment:
I've never seen the array structure for multiple conditions before. That's cool.